src/osd/PrimaryLogPG.h

```diff
@@ -347,6 +348,10 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
     ObcLockManager &manager) override {
     if (is_missing_object(hoid))
       return false;
+
+  bool try_lock_for_read(
+    const hobject_t &hoid,
+    ObcLockManager &manager) override {
```

Comment:
@shinobu-x This isn't good. These three lines you added are repeated 5 lines up.